test/index.test.ts

```diff
@@ -55,3 +58,21 @@ describe.each(allTypoSets)('%s', (name, data) => {
 		expect(contextDisplays, name).toMatchObject(data.displays);
 	});
 });
+
+describe('findConfig', () => {
+	test('should find a mock config file in working directory', async () => {
+		const readFileSpy = vi.spyOn<typeof fs, 'readFile'>(fs, 'readFile');
```

Comment:
`spyOn` might not be the best approach, because it's meant to see if a function has been called, how many times, with what params, etc, which the test isn't doing. It works, so "don't fix what's not broken" I guess, but give me a few minutes to see if I can find a more fitting solution. Otherwise, let's keep this, but the generics are currently redundant:

```suggestion
		const readFileSpy = vi.spyOn(fs, 'readFile');
```

Reply:
Looks like `spyOn` is a good way to go, since `vi.mock` only works if `fs` is imported (cosmiconfig `require`s it).

Reply:
Removed the generics here 👆.